test/passes/Oz_fuzz-exec_all-features.txt

```diff
@@ -159,14 +158,10 @@
      (i32.const 1)
     )
     (drop
-     (br_on_cast_fail $any $extendedstruct
+     (br $any
```

Comment:
Does this fail because extendedstruct is not a subtype of struct? That might be a longstanding bug in the testcase if so. But might not be worth changing.

Reply:
Yes, it would appear so. I guess we never updated it when we switched away from structural subtyping. I'll fix it.